### PR TITLE
G11 declaration content updates

### PR DESCRIPTION
**Pull request by katstevens**
Trello: https://trello.com/c/x5GVLSkq/384-implement-changes-for-legislation-changes-in-application-questions

All dates still TBC by CCS.

New questions and content updates approved by Stefan.

### CODE REVIEW

File: frameworks/g-cloud-11/questions/declaration/modernSlaveryReportingRequirements.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Modern slavery reporting requirements
+question: Does your organisation comply with the annual reporting requirements of section 54 of The Modern Slavery Act (2015)?
+
+hint: >
+  You must publish a slavery and human trafficking statement to comply the act. Read the Home Office guidance for information about [how to write a statement and what information it must include](/government/publications/transparency-in-supply-chains-a-practical-guide).
```
Comment (lfdebrux) [inline frameworks/g-cloud-11/questions/declaration/modernSlaveryReportingRequirements.yml]:
```suggestion
  You must publish a slavery and human trafficking statement to comply with the act. Read the Home Office guidance for information about [how to write a statement and what information it must include](/government/publications/transparency-in-supply-chains-a-practical-guide).
```

File: frameworks/g-cloud-11/questions/declaration/mitigatingFactors3.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Mitigating factors for Modern slavery reporting requirements
```
Comment (lfdebrux) [inline frameworks/g-cloud-11/questions/declaration/mitigatingFactors3.yml]:
```suggestion
name: Mitigating factors for modern slavery reporting requirements
```

File: frameworks/g-cloud-11/questions/declaration/servicesDoNotInclude.yml [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
-    - services that aren’t cloud-related
     - recruitment or contractor (contingent labour) services or services that are ’inside IR35’
-    - ‘colocation’ services, for example equipment the buyer rents from a supplier’s datacentre
+    - ‘colocation’ services, for example space the buyer rents from a supplier’s datacentre
     - hardware
     - bespoke design or development
 
+hint: >
+  Answer ‘yes’ to confirm your services are cloud-related and are not any of the other services described.
+  Answer ‘no’ if your service aren’t cloud related or they are one of the other services described.
```
Comment (lfdebrux) [inline frameworks/g-cloud-11/questions/declaration/servicesDoNotInclude.yml]:
```suggestion
  Answer ‘no’ if your service aren’t cloud related or if they are one of the other services described.
```

File: frameworks/g-cloud-11/questions/declaration/unfairCompetition.yml [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+  Please confirm that no party involved in the application has:
 
     - colluded to set prices with other suppliers or buyers
     - entered into any arrangement with another person so that person refrains from submitting an application
     - given access to information relating to the application to another person, or offered or given payment to another person in exchange for information
     - given inducement or valuable consideration to another person (except where such communication takes place with people who are also participants in the application, or where disclosure is made in confidence to obtain necessary information or security for the application)
     - canvassed, or attempted to buy information from, any government agent about this procurement
 
+hint: >
+    Answer ‘yes’ to confirm they have not done anything described in the question.
```
Comment (lfdebrux) [inline frameworks/g-cloud-11/questions/declaration/unfairCompetition.yml]:
```suggestion
    Answer ‘yes’ to confirm that no party involved in the application has done anything described in the question.
```

File: frameworks/g-cloud-11/questions/declaration/unfairCompetition.yml [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
+    Answer ‘no’ if they have done anything described in the question.
```
Comment (lfdebrux) [inline frameworks/g-cloud-11/questions/declaration/unfairCompetition.yml]:
```suggestion
    Answer ‘no’ if any party involved in the application has done anything described in the question.
```

File: frameworks/g-cloud-11/questions/declaration/modernSlaveryReportingRequirements.yml (hunk 1 above)
Comment (lfdebrux) [reply frameworks/g-cloud-11/questions/declaration/modernSlaveryReportingRequirements.yml]: This suggestion is the only one I think is strictly necessary.

File: frameworks/g-cloud-11/questions/declaration/mitigatingFactors3.yml (hunk 2 above)
Comment (katstevens) [reply frameworks/g-cloud-11/questions/declaration/mitigatingFactors3.yml]: I'll capitalise both - `Modern Slavery`

**Pull request merged**